include/libzfs.h

```diff
@@ -261,7 +262,8 @@ _LIBZFS_H boolean_t zpool_skip_pool(const char *);
 _LIBZFS_H int zpool_create(libzfs_handle_t *, const char *, nvlist_t *,
     nvlist_t *, nvlist_t *);
 _LIBZFS_H int zpool_destroy(zpool_handle_t *, const char *);
-_LIBZFS_H int zpool_add(zpool_handle_t *, nvlist_t *);
+_LIBZFS_H int zpool_add(zpool_handle_t *, nvlist_t *,
+    boolean_t validate_ashift);
```

Comment:
Just wonder why it is called validate_ashift here and ashift_check in other places?

Reply:
Thanks, I thought I changed them all.